views/partials/_banner.njk

```diff
@@ -0,0 +1,25 @@
+{% from "phase-banner/macro.njk" import govukPhaseBanner %}
+
+{% if PULL_REQUEST %}
+  {# Extract the PR number from Netlify's DEPLOY_PRIME_URL, which looks like
```

Comment:
I've sent an email to Netlify to see if there's a nicer way to do this – or indeed if they could add a nicer way of doing this.